Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -827,7 +1000,14 @@
       }
     },
     "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum." : {
-
+      "localizations" : {
+        "hi" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "लोरेम इप्सम केवल मुद्रण और टाइपसेटिंग उद्योग का अस्थायी पाठ है। लोरेम इप्सम 1500 के दशक से ही उद्योग का मानक अस्थायी पाठ रहा है, जब एक अज्ञात प्रिंटर ने एक प्रकार की गैली ली और उसे एक प्रकार की नमूना पुस्तक बनाने के लिए तैयार किया। यह न केवल पाँच शताब्दियों तक जीवित रहा है, बल्कि इलेक्ट्रॉनिक टाइपसेटिंग में भी छलांग लगाता रहा है, मूलतः अपरिवर्तित रहा है। इसे 1960 के दशक में लोरेम इप्सम अंशों वाले लेट्रासेट शीट के रिलीज के साथ लोकप्रिय बनाया गया था, और हाल ही में लोरेम इप्सम के संस्करणों सहित एल्डस पेजमेकर जैसे डेस्कटॉप प्रकाशन सॉफ्टवेयर के साथ इसे लोकप्रिय बनाया गया था।"
+          }
+        }
+      }
```

Comment:
Don't translate this, it's going to be removed, and is just filler text

Reply:
Sure, I'll revert this once you can confirm the other change as well.

Reply:
After you've made the updates click the re-review button please

<img width="345" alt="Re-review button" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/22946902/52982f6c-d50b-4f59-93a3-f7df8716d39a">

Reply:
Done, I've removed the unused translation.